inputs/attached-button.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="kiln-attached-buttons ui-button-group" v-if="args.attachedButton">
+    <component v-for="button in buttons" :key="button.name" slot="icon" :is="button.name" :name="name" :data="data" :schema="schema" :args="button" @disable="onDisable" @enable="onEnable"></component>
+  </div>
+</template>
+
+<script>
+  import _ from 'lodash';
+  import logger from '../lib/utils/log';
+
+  const log = logger(__filename);
+
+  export default {
+    name: 'attached-button',
+    props: ['name', 'data', 'schema', 'args'],
+    data() {
+      return {};
+    },
+    computed: {
+      buttons() {
+        const attachedButton = _.get(this, 'args.attachedButton');
+
+        let buttons;
+
+        // munge them into an array
+        if (!!attachedButton) {
+          buttons = _.isArray(attachedButton) ? attachedButton : [attachedButton];
```

Comment:
maybe add `log.warn` if there are more than 2 buttons?

Reply:
Eh, it's not recommended in most cases (because it makes the input smaller), but there's nothing actually wrong with using more than two attached buttons